nikola/plugins/command/new_post.py

```diff
@@ -213,6 +213,14 @@ class CommandNewPost(Command):
                     'bbcode, html, textile, txt2tags',
         },
         {
+            'name': 'kernel',
```

Comment:
Just an `ipynb` with a different kernel spec... the thing about supporting more than the 4 main ones is that we need to explore how each of the other ~30 kernels specify their kernelspec and be awared of any change in the future...